softdev.bib

```diff
@@ -2502,3 +2502,12 @@ @inproceedings{prokopec19renaissance
     month = Jun,
     doi = "10.1145/3314221.3314637",
 }
+
+@inproceedings{blazakis10interpreter,
+    author = "Dionysus Blazakis",
+    title = {Interpreter Exploitation},
+    booktitle = "WOOT",
+    year = {2010},
```

Comment:
I'm not sure if it matters or even what our default is, but the other papers above don't put curlys around the year.

Reply:
Good spot. Fixed in https://github.com/softdevteam/softdevbib/pull/58/commits/6e29d8af22995285c2f9e11b07ab6d401e2ed531